migrations/0xx.construction-materials.up.sql

```diff
@@ -0,0 +1,38 @@
+-- Launch of Constuction category
+-- Fields: Core construction material, Secondary construction materials, Roof covering materials
+
+-- Construction materials: Wood, Stone, Brick, Steel, Reinforced Concrete, Other Metal
+--   Other Natural Material, Other Man-Made Material
+CREATE TYPE construction_materials
+    AS ENUM ('Wood',
+    'Stone',
+    'Brick',
+    'Steel',
+    'Reinforced Concrete',
+    'Other Metal',
+    'Other Natural Material',
+    'Other Man-Made Material');
+
+-- Roof covering materials: Slate, Clay Tile, Wood, Asphalt, Iron or Steel, Other Metal
+--   Other Natural Material, Other Man-Made Material
+CREATE TYPE roof_covering
+    AS ENUM ('Slate',
+    'Clay Tile',
+    'Wood',
+    'Asphalt',
+    'Iron or Steel',
+    'Other Metal',
+    'Other Natural Material',
+    'Other Man-Made Material');
+
+-- Core Construction Material
+ALTER TABLE buildings
+    ADD COLUMN IF NOT EXISTS core_materials construction_materials;
+
+-- Secondary Construction Materials
+ALTER TABLE buildings
+    ADD COLUMN IF NOT EXISTS secondary_materials construction_materials;
```

Comment:
Do we intend this to hold just one value? Secondary materials seems like something which could potentially hold multiple values (as opposed to core material) but then we need to change the column type to array of `construction_material`, which then necessitates several changes elsewhere in the code. Not sure what is the plan here.
If singular is ok, then I'd also change the column name to singular

Reply:
Secondary materials should hold more than one value but I couldn't figure out how to create a drop down list than allows multiple selections. Any suggestions?

Reply:
Secondary materials should hold more than one value but I couldn't figure out how to create a drop down list than allows multiple selections. Any suggestions?

Reply:
For the database side, the type of the column needs changing to an array type: `construction_material[]` 

The UI bit will be a `MultiDataEntry` component, but the latest version of that is on the land use branch so @tomalrussell needs to merge that one into master first. I thought we were going to just add the core material for construction, which is why we didn't go through that in the workshop. Currently the MultiDataEntry allows for a multi-valued list of free text fields, with an optional auto-fill dropdown which works by contacting the server the get the list of options based on typed text. There isn't currently an option for a pure dropdown option so that gets complicated. An idea for @tomalrussell: could change MultiDataEntry to accept any type of single data entry, for example through a render prop? So that it can be a list of dropdowns, list of text fields, list of text fields with autofill etc.

The amount of work is hard to estimate for the above - might be easiest to release just the single-valued core material and roof material, and then develop the secondary materials later?

Reply:
Yes - this is a good note. I'll merge this as-is and see if it's straightforward to build on top with `MultiDataEntry` before releasing.